shared/helpers/tests.js

```diff
@@ -40,6 +40,20 @@ async function compileSassString (source, options = {}) {
   })
 }
 
+/**
+ * Fetch by URL request path
+ *
+ * @param {string} path - URL request path
+ * @param {RequestInit} [options] - Fetch options
+ * @returns {Promise<Response>} Fetch response
+ */
+const fetchPath = (path, options = {}) => {
+  return fetch(new URL(path, `http://localhost:${ports.app}`), {
+    keepalive: true,
```

Comment:
Using `{ keepalive: true }` appears to avoid [`other side closed` errors in Node.js](https://github.com/nodejs/node/blob/7196946d7f59652c19dec278186165806b1e28ad/deps/undici/src/lib/client.js#L989-L995)